packages/core/src/Stores/notification-store.js

```diff
@@ -341,6 +341,9 @@ export default class NotificationStore extends BaseStore {
 
             this.handlePOAAddressMismatchNotifications();
 
+            // TODO: Update logic when BE API is integrated [Wallets]
+            // this.showSuccessWalletsUpgradeNotification();
```

Comment:
@sergei-deriv Why these lines are added here?

Reply:
@mahdiyeh-deriv it's Aizad's code which I've commented but I think it was deleted in the feature branch. I'll delete this